### PR TITLE
[OMA-815] - HyBid remote config models

### PR DESCRIPTION
**Pull request by PabloGarciaAldehuela**

### CODE REVIEW

File: hybid.sdk/src/main/java/net/pubnative/lite/sdk/models/RemoteConfigVoyager.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    public RemoteConfigVoyager(JSONObject jsonObject) throws Exception {
+        fromJson(jsonObject);
+    }
+
+
+
+
+
+
+    //todo need to create some similar find method?
```
Comment (eros902002) [inline hybid.sdk/src/main/java/net/pubnative/lite/sdk/models/RemoteConfigVoyager.java:38]: We can remove this commented out method I think.

File: hybid.sdk/src/main/java/net/pubnative/lite/sdk/models/RemoteConfigVoyagerData.java [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+import net.pubnative.lite.sdk.utils.json.BindField;
+import net.pubnative.lite.sdk.utils.json.JsonModel;
+
+import org.json.JSONObject;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class RemoteConfigVoyagerData extends JsonModel {
```
Comment (eros902002) [inline hybid.sdk/src/main/java/net/pubnative/lite/sdk/models/RemoteConfigVoyagerData.java:11]: We don't need this class anymore

Comment (PabloGarciaAldehuela) [reply hybid.sdk/src/main/java/net/pubnative/lite/sdk/models/RemoteConfigVoyagerData.java:11]: Deleted

File: hybid.sdk/src/main/java/net/pubnative/lite/sdk/models/RemoteConfigVoyager.java (hunk 1 above)
Comment (PabloGarciaAldehuela) [reply hybid.sdk/src/main/java/net/pubnative/lite/sdk/models/RemoteConfigVoyager.java]: Deleted

File: hybid.sdk/src/main/java/net/pubnative/lite/sdk/models/RemoteConfigAudience.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+    }
+
+    public RemoteConfigAudience(JSONObject jsonObject) throws Exception {
+        fromJson(jsonObject);
+    }
+
+
+    public Integer getDistanceThreshold(){
+        return (Integer) getParamsField("distance_threshold");
```
[the remark this answers is not in the document]
Comment (PabloGarciaAldehuela) [reply hybid.sdk/src/main/java/net/pubnative/lite/sdk/models/RemoteConfigAudience.java]: Added check

**Review by eros902002 — APPROVED** [review]

**Pull request merged**